packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -25,6 +25,13 @@ const mockFlags: { [key: string]: any } = {
   estimateGas: null,
 };
 
+jest.mock('uuid', () => {
```

Comment:
Another case where we are stubbing `uuid` as a replacement for the global mock in `__mocks__`, to make `uuid.v1()` deterministic.